src/backend/metal/src/device.rs

```diff
@@ -283,6 +319,15 @@ impl PhysicalDevice {
                 format_bc: Self::is_mac(device),
                 format_eac_etc: !Self::is_mac(device),
                 format_astc: Self::supports_any(device, ASTC_PIXEL_FORMAT_FEATURES),
+                format_r8unorm_srgb_all: Self::supports_any(device, R8UNORM_SRGB_ALL),
```

Comment:
could this fit into the old "format_min_srgb_channels" property?

Reply:
@kvark Not exactly, there are some versions of iOS and tvOS that don't support it.